src/kernel/arch/aarch64/arch.zig

```diff
@@ -120,5 +109,133 @@ pub fn haltNoInterrupts() noreturn {
 
 // TODO: implement
 pub fn spinWait() noreturn {
-    while (true) {}
+    spinLed(500);
```

Comment:
I think this should still be a while true loop (it should probably include a `wfe` but that can wait) since you won't always want to make the LED flash when spinning.

Reply:
Done: 313b85716e147c5a5393fb938d40fedf4932d8f4